### PR TITLE
Fix IDictionary tests

### PR DESCRIPTION
**Pull request by isc30**
The current implementation doesn't allow implementing a custom `IDictionary`, but we are close.
This PR covers the already implemented functionality, in order to avoid breaking it.

### CODE REVIEW

File: src/DSharp.Mscorlib/Collections/IDictionary.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public interface IDictionary : IEnumerable
         [DSharpScriptMemberName("keyExists")]
         bool Contains(object key);
 
+        [DSharpScriptMemberName("addKeyValue")]
```
Comment (fred-perkins) [inline src/DSharp.Mscorlib/Collections/IDictionary.cs]: Can you make sure this method actually exists in the ss script, I remember recently trying this and it wasn't working.

Comment (isc30) [reply src/DSharp.Mscorlib/Collections/IDictionary.cs]: done :)

**Comment by isc30** [comment]
Let's get this merged, please

**Comment by fred-perkins** [comment]
Merged and released as https://github.com/DerivcoIpswich/dsharp/releases/tag/1.3.3

**Pull request merged**